braindecode/datautil/windowers.py

```diff
@@ -176,7 +176,7 @@ def create_fixed_length_windows(
     for ds in concat_ds.datasets:
         # already includes last incomplete supercrop start
         stop = (ds.raw.n_times
-                if stop_offset_samples is None
+                if (stop_offset_samples is None or stop_offset_samples == 0)
```

Comment:
why this == 0 here makes sense? can you add a small comment maybe?

Reply:
Comment here or in the code you mean?
`stop_offset_samples` can be unset (`None`) or `int` (including 0). Both, `None` and 0 have the same meaning: there is no stop offset.

Reply:
Alternatively, instead of adding this here, one could adjust the checks (https://github.com/braindecode/braindecode/blob/baa86663b8765207b4f4a744661ce6fdc4a78da0/braindecode/datautil/windowers.py#L294-L295) of the windowing arguments to only allow integer values > 0 for `stop_offset_samples` which probably makes more sense. What do you think?